docs/faq.md

```diff
@@ -158,15 +158,16 @@ Chromebooks will need higher requirements if you're using Linux (Beta)/Crostini.
 These specs will provide full-speed gameplay almost everywhere, with some slowdowns in non-critial areas with a large amount of objects, such as the main caverns of Area 2, Area 3, and Area 5. Weaker GPUs may slow down in Fusion difficulty during X Parasite morphing.
 
 #### Where can I stay up-to-date on AM2R?  
-<https://bit.ly/AM2Rblog>  
-<https://reddit.com/r/AM2R>  
-<https://discord.gg/YTQnkAJ>  
-<https://youtube.com/c/am2rcommunityupdates>
+- [Reddit](https://reddit.com/r/AM2R)
+- [Discord](https://discord.gg/YTQnkAJ) and [Matrix](https://matrix.to/#/#am2r:matrix.org). Discord and Matrix are bridged together.
+- [Community Updates YouTube Channel](https://youtube.com/c/am2rcommunityupdates)
+- [Legacy AM2R blog](https://metroid2remake.blogspot.com/) (No community updates will be present here)
 
 #### Can I play AM2R on a Mac?  
-Macs are not officially supported. There is however, an unofficial port available here: <https://github.com/Miepee/AM2R-Autopatcher-Mac>  
+Macs are not officially supported. There is however, an unofficial port available here: <https://github.com/Miepee/AM2R-Autopatcher-Mac>
```

Comment:
Whitespace at the end is missing, which breaks formatting of the next paragraph.

Reply:
Fixed and reviewed in preview.